src/modeliteration.js

```diff
@@ -130,7 +130,8 @@ define([
                     presence: 'optional',
                     widget: idsQueryDataSourceWidget,
                 }
-            }
+            },
+            noLogicReference: true
```

Comment:
When the `modeliteration` plugin is enabled this `modelRepeatMugOptions` object applies to all Repeat questions. How does this not disable `repeat_count` validation on all Repeat questions?

I see you have added tests to verify that it is/is not working at the correct times, but I can't figure out why they're passing. ... ok, on further inspection I think I get it. This disables the validation error message (for all Repeat questions), but it does not disable the automatic reference updates. So, unlike other mugs, there will be no validation error if you delete the node that a regular Repeat question's `repeat_count` references. If you wanted to fix that issue you could make `noLogicReference` a function instead of a constant. Something like this:
```js
noLogicReference: function (mug) {
    return !mug.p.dataSource.idsQuery;
}
```